packages/server/utils/tasks/account/wallet.js

```diff
@@ -7,11 +7,13 @@ const fs = require('fs');
 const path = require('path');
 
 const arweave = init();
-const keyPath = path.join(__dirname,"..", "key.json");
+const keyPath = path.join(__dirname, ".." ,"..", "..", "secrets.json");
```

Comment:
this doesn't seem very elegant...

Reply:
i don't think it matters alot, since the secrets.json will just live in the root of the server directory and this particular script just creates a random key and stores in there